ui/app/pages/swaps/swaps.util.js

```diff
@@ -35,13 +35,18 @@ const getBaseApi = function (type) {
 
 const validHex = (string) => Boolean(string?.match(/^0x[a-f0-9]+$/u))
 const truthyString = (string) => Boolean(string?.length)
+const stringIsValidDecimalNumber = (string) => !isNaN(string) && string.match(/^[.0-9]+$/u) && !isNaN(parseFloat(string))
 const truthyDigitString = (string) => truthyString(string) && Boolean(string.match(/^\d+$/u))
 
 const QUOTE_VALIDATORS = [
   {
     property: 'trade',
     type: 'object',
-    validator: (trade) => trade && validHex(trade.data) && isValidAddress(trade.to) && isValidAddress(trade.from) && truthyString(trade.value),
+    validator: (trade) => trade &&
```

Comment:
Just for clarity, do we want to add this as a function like `stringIsValidDecimalNumber` is on line 38?  Maybe `const tradeIsValid = ....`.  Maybe just for clarity.